po/clubhouse.pot

```diff
@@ -0,0 +1,175 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the clubhouse package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: clubhouse\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-11-05 13:25+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/achievement-summary-view.ui:33
+msgid "Achievement Badge Name"
+msgstr ""
+
+#: data/achievement-summary-view.ui:49
+msgid "Description"
+msgstr ""
+
+#: data/achievements-view.ui:66
+msgid "Achievements"
+msgstr ""
+
+#: data/achievements-view.ui:131
+msgid "page0"
+msgstr ""
+
+#: data/achievements-view.ui:163
+msgid "page1"
+msgstr ""
+
+#: data/activity-card.ui:219
+msgid "play"
+msgstr ""
+
+#: data/character-view.ui:81 data/clubhouse-window.ui:255
+#: data/clubhouse-window.ui:306
+msgid "Clubhouse"
+msgstr ""
+
+#: data/character-view.ui:124
+msgid "Workspace"
+msgstr ""
+
+#: data/clubhouse-view-main-layer.ui:102
+msgid "Hack extension is not enabled. Click here to solve"
```

Comment:
This label is missing the last period "Click here to solve.". Also it would be better to have a native speaker review all labels. Might be better something like: "The Hack extension is not enabled. Please click here to enable it.".